helpers/delegate-frontend/contracts/DelegateFrontend.sol

```diff
@@ -166,7 +166,7 @@ contract DelegateFrontend {
     IERC20(_signerToken).approve(address(swapContract), signerAmount);
 
     // DelegateFrontend authorizes the Delegate.
-    swapContract.authorize(delegateContract, block.timestamp + 1);
+    swapContract.authorizeSigner(delegateContract, block.timestamp + 1);
```

Comment:
Alice is not signing the order, the delegatefrontend is authorizing the delegateContract to bypass the signature component by being the authorized signer on the order.

Reply:
Thus the delegateFront needs to call an authorizeSigner rather than authorizeSender. The tradeWallet has made the delegate the authorizedSender for itself though.

Reply:
Okay makes sense. Am I right in thinking this would be in conjunction with Bob using `authorizeSender` for the delegate?
i.e.
- delegateFrontEnd calls `authorizeSigner` for Delegate as Alice isn't signing
- Bob calls `authorizeSender` for Delegate as Bob isn't Sending

Reply:
yes, Bob (tradeWallet) needs to authorize this own delegate to be the authorizedSender.